assets/common/src/register.py

```diff
@@ -128,16 +107,17 @@ def get_ml_client(registry_name):
         has_obo_succeeded = True
```

Comment:
nit: since we are using common src now, could we move similar functionality in utils file? for instance get_ml_client()

Reply:
Can we do this changes in next PR. lets go with logs detail for this pr